core.js

```diff
@@ -2170,10 +2170,6 @@ class HTMLScriptElement extends HTMLLoadableElement {
     this._emit('innerHTML', innerHTML);
   }
 
-  set textContent(textContent) {
```

Comment:
I still don't think this is right, since setting the src probably won't run the script now. Should it?

What does the spec say?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement

![image](https://user-images.githubusercontent.com/59632/39838235-68eec714-539e-11e8-861f-8a4edfea57bf.png)

Chrome doesn't re-execute scripts that mutate via .textContent, so I think this PR is ready to merge.